src/main/scala/coreplex/Configs.scala

```diff
@@ -22,7 +22,7 @@ class BaseCoreplexConfig extends Config ((site, here, up) => {
   case BuildCore => (p: Parameters) => new Rocket()(p)
   case RocketCrossing => Synchronous
   case RocketTilesKey =>  Nil
-  case DMKey => new DefaultDebugModuleConfig(site(XLen))
+  case DMKey => DefaultDebugModuleConfig(site(XLen))
```

Comment:
That's not quite what the DefaultDebugModuleConfig is doing, it actually takes in XLEN as (currently the only) parameter and then turns it into nDMIAddrSize. Point well taken though